docs/_documentations/linking-your-projects.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Linking your projects
+description: Linking your projects
+keywords: project actions, attach, build, disable, enable, validate, refresh, link, linking your projects, VS Code, Eclipse
+duration: 1 minute
+permalink: linking-your-projects.html
+type: document
+---
+
+# Linking your projects
+
+You can link your projects when you are using Codewind. This feature is available in the [VS Code](#linking-your-projects-in-vs-code) and [Eclipse](#linking-your-projects-in-eclipse) IDEs.
+
+For example, use this feature to link a front-end Node.js project to a back-end database. You have a web page front-end Node.js project without a link:
+
+![Image of unlinked project](images/linking-feature/unlinked_project.png){:width="650px"}
+
+You then link it to a back-end Node.js database project:
+
+![Image of linked project](images/linking-feature/linked_project.png){:width="650px"}
+
+## Linking your projects in VS Code
+
+To link your projects in VS Code, follow these steps where project `frontend-nodejs` is the source project and project `database-nodejs` is the target project. Both projects must be running to create a link.
+
+1\. Right-click project `frontend-nodejs` in the **Codewind** view, select **Project Overview**, and then select the **Links** tab.
+
+2\. Click **Create Link**:
+
+![Image of managing VS Code project links](images/linking-feature/vscode_add_project_link.png){:width="650px"}
+
+3\. Select the target project and press **Enter**. 
+
+![Image of selecting VS Code target project](images/linking-feature/vscode_select_target_project.png){:width="650px"}
+
+4\. Enter a name for the environment variable and press **Enter**:
+
+![Image of entering VS Code environment variable](images/linking-feature/vscode_enter_environment_variable.png){:width="650px"}
+
+5\. The link is created and is shown in the **Project Links** section of the **Project Overview**:
+
+![Image of VS Code project links section](images/linking-feature/vscode_manage_project_links_step_4.png){:width="650px"}
+
+6\. Use the environment variable in your code as needed, for example:
+
+`axios.get(http://${process.env.DATABASE}/api/v1/populations);`
```

Comment:
Thinking about the resolved comment, this could be changed to:
```javascript
// Where process.env.DATABASE is the name of the environment variable added in step 4
axios.get(http://${process.env.DATABASE}/api/v1/populations);
```

Reply:
Have changed the { } to < > according to our usual notation, and have added the sentence.